flow-nodes/api-builder-plugin-logger/src/flow-nodes.yml

```diff
@@ -21,20 +23,23 @@ flow-nodes:
                 - error
               default: trace
           message:
+            name: Logmessage
```

Comment:
Why didn't you use a space?  `Logmessage => Log message`

Reply:
Resolved now.